orc8r/cloud/docker/controller/Dockerfile

```diff
@@ -27,6 +27,9 @@ RUN curl https://facebookconnectivity.jfrog.io/artifactory/generic/go1.13.4.linu
     tar xf go1.13.4.linux-amd64.tar.gz && \
     cp -r go/bin/* /usr/local/bin/
 
+# Install goimports
+RUN go get golang.org/x/tools/cmd/goimports
```

Comment:
if you add `golang.org/x/tools/cmd/goimports` to the `TOOL_DEPS` below you will get the same result i think


https://sourcegraph.com/-/editor?remote_url=git%40github.com%3Auri200%2Fmagma.git&branch=master&file=orc8r%2Fcloud%2Fgo%2FMakefile&editor=JetBrains&version=v1.2.1&utm_product_name=IntelliJ+IDEA&utm_product_version=2021.2.2&start_row=1&start_col=0&end_row=1&end_col=9

Reply:
we have the same for feg btw

https://sourcegraph.com/-/editor?remote_url=git%40github.com%3Auri200%2Fmagma.git&branch=master&file=feg%2Fgateway%2FMakefile&editor=JetBrains&version=v1.2.1&utm_product_name=IntelliJ+IDEA&utm_product_version=2021.2.2&start_row=98&start_col=0&end_row=98&end_col=0

Reply:
added it to make as well, we can remove redundant part later